### PR TITLE
Print coordinates of cells where flux correction is needed

### LINKED ISSUE #381: Print coordinates of cells requiring flux correction
Can we also print the (x,y,z) coordinates of cells that require flux correction, in addition to their State variable contents:

https://github.com/quokka-astro/quokka/blob/ca3f360580ba184058ac696aadeedb600e94ec72/src/RadhydroSimulation.hpp#L1027

### PR DESCRIPTION
**Pull request by psharda**
Closes #381

### CODE REVIEW

File: src/RadhydroSimulation.hpp [hunk 1]
```diff
@@ -1026,6 +1026,18 @@ auto RadhydroSimulation<problem_t>::advanceHydroAtLevel(amrex::MultiFab &state_o
...
+				amrex::Real y_coord = NAN;
+				amrex::Real z_coord = NAN;
+				if (AMREX_SPACEDIM > 1) {
+					y_coord = geom[lev].ProbLo(1) + (cell_idx[1] + 0.5) * geom[lev].CellSize(1);
+					if (AMREX_SPACEDIM > 2) {
+						z_coord = geom[lev].ProbLo(2) + (cell_idx[2] + 0.5) * geom[lev].CellSize(2);
+					}
+				}
+
+				amrex::Print() << x_coord << ", " << y_coord << ", " << z_coord << ": ";
```
Comment (BenWibking) [inline src/RadhydroSimulation.hpp]: Can you put this in a separate function? I think that would be better than duplicating the code.

**Review by BenWibking — DISMISSED** [review]

**Review by BenWibking — APPROVED** [review]

**Comment by psharda** [comment]
ugh, avatarGPUs throw `nvcc error   : 'cicc' died due to signal 9 (Kill signal)`

**Comment by BenWibking** [comment]
Somehow the system ran out of memory when it was trying to compile. Maybe try again?

**Pull request merged**